generic/tests/dd_test.py

```diff
@@ -173,5 +174,6 @@ def _check_disk_partitions_number():
         test.log.info("dd test succeeded.")
     finally:
         for dev_id in dev_partitioned:
```

Comment:
It is strange exist two sessions but without any comment, suggest solution：
could you please try login again before the clean operation
finally:
 # login again in case the previous session expired
session = vm.wait_for_login(timeout=timeout)
 for dev_id in dev_partitioned:
            utils_disk.clean_partition_linux(session, dev_id)

Reply:
Updated.